### DISCUSSION TITLE
提示：出现身份验证错误，如何解决

### DISCUSSION THREAD

**Discussion by WENQINGHAN** [natfrp/wiki#204] [使用问题]
情况描述：10月15号上午可以登陆，下午登的时候出现身份验证错误，看通告说因为某些原因预计27号可以登录，但是27号至今还是出现同样错误。
错误描述：发生身份验证错误（代码：0x80004005) 远程计算机：xxxxxxxxxxx.natfrp.cloud
经过某些尝试（Firewll等等的尝试），问题未解决

**Comment by fakeboboliu** [comment]
您好，
机房仍然在封禁我们机器的 TLS 流量，
这将导致使用 CredSSP 的 RDP 流量也遭牵连，我们建议您在控制面板将隧道迁移到非枣庄节点。
